### ISSUE TITLE
Tizen 4 (Samsung TV 2018) - Error 3016 with Sintel 4k (multicodec, Widevine)

### ISSUE THREAD

**Issue by avelad** [google/shaka-player#1751]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**: Yes

**What version of Shaka Player are you using?**: 2.5.0-beta2

**Can you reproduce the issue with our latest release version?**: Yes

**Can you reproduce the issue with the latest code from `master`?**: Yes

**Are you using the demo app or your own custom app?**: Both

**If custom app, can you reproduce the issue using our demo app?**: Yes

**What browser and OS are you using?**:
Tizen 4 (Samsung TV 2018) 
Samsung UE43NU7405

**What are the manifest and license server URIs?**:

manifestUri: 'https://storage.googleapis.com/shaka-demo-assets/sintel-widevine/dash.mpd'
'com.widevine.alpha': 'https://cwip-shaka-proxy.appspot.com/no_auth',

**What did you do?**
Load previous stream

**What did you expect to happen?**
There are not errors

**What actually happened?**
Nothing else to start shows the error 3016 (browser error 3)

**Comment by chrisfillmore** [comment]
Do you know what variant is playing (or trying to play) when you get the error?

I noticed there are a few AdaptationSets for video, with a variety of codecs. Some of these Tizen may not actually support (even if they claim they do). I would recommend removing variants from the manifest until it plays for you, then find the problematic codecs.

Otherwise, this is just an error from the Tizen media stack and there's nothing we can know about it. :(

**Comment by vaage** [comment]
That is a great question @chrisfillmore, thanks for the help! @avelad do you know what codecs your content is trying to use?

**Comment by avelad** [comment]
https://storage.googleapis.com/shaka-demo-assets/sintel-widevine/dash.mpd
Using:
video codec: vp9
audio codec: opus 
subtitle: text/vtt

**Comment by vaage** [comment]
@chrisfillmore do you know if any of those codec normally cause issues with the Tizen media stack?

@avelad sorry for the delay in catching this, I should have followed-up on this sooner.

**Comment by chrisfillmore** [comment]
@vaage Unfortunately I don't know. We don't use those codecs in our project, so far as I'm aware.

**Comment by avelad** [comment]
I just checked with .mkv with vp9 and opus and it works

**Comment by vaage** [comment]
So it sounds like the media stack does not have any issues with the container+codecs. The next question is, "what is different when we use media source to play the content?".

Regardless of if it's a good assumption, I am assuming the content ([https://storage.googleapis.com/shaka-demo-assets/sintel-widevine/dash.mpd](https://storage.googleapis.com/shaka-demo-assets/sintel-widevine/dash.mpd)) is not the problem since we use it everywhere.

What I want to look at next is, what happens if we only play one streams (only-audio then only-video) with media source. To help isolate the points of failure, I think it would be best to use a stripped-down media source example.

I will create the test case and reply with it so that @avelad can test which streams fail. If one of the streams fail with the same error, we will have a much more accurate understand of what is involved with the failure. If we can't reproduce the error, than it means that it is more than just a problem with media source on Tizen.

@avelad, what are your thoughts?

**Comment by avelad** [comment]
In my opinion, i think that the problem is related with the MSE implementation with these codecs. I just tested with VP9 and Vorbis and all it's fine.

**Comment by vaage** [comment]
@avelad How did you conduct your tests? I want to make sure I understand correctly because with it sounds like:

```
PLAYBACK METHOD | VIDEO CODEC | AUDIO CODEC | RESULT
----------------+-------------+-------------+-------
MSE             | VP9         | opus        | FAIL
SRC=            | VP9         | opus        | PASS
MSE             | VP9         | vorbis      | PASS
```

**Comment by joeyparrish** [comment]
I will go ahead and mark this as a bug since we intend to work around it.  In truth, it seems to be Samsung's bug.

**Comment by joeyparrish** [comment]
Our Tizen TV is inaccessible right now, and may remain so for a couple days.  We have a speculative fix, but haven't been able to verify it yet.  I will also go ahead and submit the change, though.  @avelad, can you please test and confirm the fix for us when you have time?

**Comment by avelad** [comment]
I'm on vacation until the 18th. On my return I can prove it!

**Comment by joeyparrish** [comment]
No worries.  I may be able to test it myself before then.

**Comment by joeyparrish** [comment]
Fix released in v2.4.7.  Let us know if you have any further issues!
